extensions/amp-list/0.1/amp-list.js

```diff
@@ -64,7 +65,11 @@ export class AmpList extends AMP.BaseElement {
           if (!opts.credentials) {
             opts.requireAmpResponseSourceOrigin = false;
           }
-          return xhrFor(this.win).fetchJson(src, opts);
+          const fetchPromise = batchedXhrFor(this.win).fetchJson(src, opts);
+          const fragment = getFragment(src).slice(1);
```

Comment:
Justin and I discussed keeping the fragment-extraction and object property access behavior inside of the new Xhr class, but we decided to keep them separate. [Discussion here.](https://github.com/ampproject/amphtml/pull/7562#discussion_r103353100)